From 3a35611687864cf3599712d0d2ff54532b0118ac Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Sun, 3 Oct 2021 21:42:32 +0200
Subject: [PATCH] m4/ax_python_devel.m4: do not check for distutils

With py 3.10 this prints a deprecation warning which is
taken as an error. Upstream should rework the code to not
use distuils.

Upstream-Status: Inappropriate [needs a proper fix upstream]
Signed-off-by: Alexander Kanavin <alex@linutronix.de>
---
 m4/ax_python_devel.m4 | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/m4/ax_python_devel.m4 b/m4/ax_python_devel.m4
index e0804c5..cfc0b24 100644
--- a/m4/ax_python_devel.m4
+++ b/m4/ax_python_devel.m4
@@ -138,21 +138,6 @@ variable to configure. See ``configure --help'' for reference.
 		fi
 	fi
 
-	#
-	# Check if you have distutils, else fail
-	#
-	AC_MSG_CHECKING([for the distutils Python package])
-	ac_distutils_result=`$PYTHON -c "import distutils" 2>&1`
-	if test -z "$ac_distutils_result"; then
-		AC_MSG_RESULT([yes])
-	else
-		AC_MSG_RESULT([no])
-		AC_MSG_ERROR([cannot import Python module "distutils".
-Please check your Python installation. The error was:
-$ac_distutils_result])
-		PYTHON_VERSION=""
-	fi
-
 	#
 	# Check for Python include path
 	#
